tests/behat/docs/Scenarios/seek012.md

```diff
@@ -0,0 +1,27 @@
+# Scenario: Seek012
+
+Check internal searchfield.
+
+Performs search, switches to internal search and checks that a result is found.
+
+## Method
+This scenario utilizes the variable-substitution functionality in
+
+**Given I have search for "$random.nyhed"**
+
+The $ is the variable-indicator, which means the method will attempt to translate the value.
+The actual variable consists of a modifier and a variable name.
+The modifier is in the example above __random__ and the variable is __nyhed__.
+The modifier could be random, first, last, but would be defined for each variable because each variable that will ever be defined would have its own context.
+Nyhed is actually equal to 'news', so both english and danish is possible. The benefit of having multiple synonyms for these is that the testers/users will use what is most natural for them. 
```

Comment:
Not sure if this is an actual advantage, or will just cause confusion. If the option to use both danish and english is kept it should be done consistently. Right now it seems random when this is implemented. E.g. `translateFieldName()` understands "søg" and "søgefelt" but not "search", while ´translateArgument()` understands both "nyhed" and "news".